public.yaml

```diff
@@ -19,12 +19,61 @@ metadata:
     name: velum-public
 spec:
   hostNetwork: true
+  initContainers:
+  - name: init-mariadb
+    image: library/mariadb:10.0.29
+    imagePullPolicy: IfNotPresent
+    env:
+    - name: MYSQL_ROOT_PASSWORD
+      value: "salt"
+    - name: MARIADB_SOCKET
+      value: /var/run/mysqld/mysqld.sock
+    command: ["/bin/sh", "-c"]
+    args: ["until mysqladmin -p\"${MYSQL_ROOT_PASSWORD}\" -S \"${MARIADB_SOCKET}\" ping; do echo 'Database not online yet, waiting...' && sleep 3; done"]
+    volumeMounts:
+      - mountPath: /var/run/mysqld
+        name: mariadb-unix-socket
+  - name: init-rake-setup
+    image: opensuse/velum:development
+    imagePullPolicy: IfNotPresent
+    env:
+    - name: RAILS_ENV
+      value: "development"
+    - name: VELUM_SECRETS_DIR
+      value: /var/lib/misc/velum-secrets
+    - name: RAILS_SERVE_STATIC_FILES
+      value: "true"
+    - name: VELUM_PORT
+      value: "3000"
+    - name: VELUM_DB_USERNAME
+      value: "root"
+    - name: VELUM_DB_PASSWORD
+      value: "salt"
+    - name: VELUM_DB_SOCKET
+      value: "/var/run/mysqld/mysqld.sock"
+    - name: VELUM_SALT_HOST
+      value: "127.0.0.1"
+    - name: VELUM_SALT_PORT
+      value: "8000"
+    - name: VELUM_SALT_USER
+      value: "saltapi"
+    - name: VELUM_SALT_PASSWORD
+      value: "saltapi"
+    command: ["/bin/sh", "-c"]
+    args: ["bundle exec rake db:create && bundle exec rake db:schema:load && bundle exec rake db:migrate"]
```

Comment:
This will break the development environment, maybe we need to split `bin/init` in https://github.com/kubic-project/velum/blob/master/bin/init, but the `db:schema:load` needs this https://github.com/kubic-project/velum/blob/feac9eb14c16d0e250feeeb2e42c73a64781e178/bin/init#L41-L43.

If we split the start script we could call that script from here, without the need to explicitly run this commands, and it would work fine in development and in production.